test/org/sagebionetworks/bridge/dynamodb/DynamoInitializerIntegrationTest.java

```diff
@@ -0,0 +1,147 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.amazonaws.services.datapipeline.DataPipelineClient;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.datapipeline.model.PutPipelineDefinitionRequest;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.sagebionetworks.bridge.config.BridgeConfig;
+import org.sagebionetworks.bridge.config.BridgeTestSpringConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+@ContextConfiguration(classes={BridgeTestSpringConfig.class})
+@RunWith(SpringJUnit4ClassRunner.class)
+public class DynamoInitializerIntegrationTest {
```

Comment:
As written, this test would create real pipelines and hit real AWS services. I feel like it would be better to just mock everything.